### ISSUE TITLE
Unhandled 'error' event

### ISSUE THREAD

**Issue by johnholliday** [kristianmandrup/generator-stenciljs#3]
```
register gadget-test with Stencil
events.js:160
      throw er; // Unhandled 'error' event
      ^

Error: Cannot find module 'D:\Temp\stencil-component\stencil.config.js'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at writing (C:\Users\John\AppData\Roaming\npm\node_modules\generator-stenciljs\generators\component\index.js:494:22)
    at Object.<anonymous> (C:\Users\John\AppData\Roaming\npm\node_modules\generator-stenciljs\node_modules\yeoman-generator\lib\index.js:399:25)
    at C:\Users\John\AppData\Roaming\npm\node_modules\generator-stenciljs\node_modules\run-async\index.js:25:25
    at C:\Users\John\AppData\Roaming\npm\node_modules\generator-stenciljs\node_modules\run-async\index.js:24:19
    at self.env.runLoop.add.completed (C:\Users\John\AppData\Roaming\npm\node_modules\generator-stenciljs\node_modules\yeoman-generator\lib\index.js:400:11)
    at runCallback (timers.js:672:20)
```

**Comment by kristianmandrup** [comment]
Yeah, sorry. Need to make a new release today. Didn't check my last release properly. Needs test suite. Working on it... Welcome to help out and make a PR to fix the bug.

**Comment by kristianmandrup** [comment]
Fixed it now, version 0.0.3. Could really use some help to improve it ;)
